multicluster/controllers/multicluster/suite_test.go

```diff
@@ -86,7 +87,13 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
-
+	out, _ := exec.Command("which", "kind").Output()
+	if len(string(out)) == 0 {
+		err := exec.Command("sudo", "../../../ci/kind/kind-setup.sh", "kind-install").Run()
+		Expect(err).NotTo(HaveOccurred())
+	}
+	err := exec.Command("sudo", "../../../ci/kind/kind-setup.sh", "create", "kind").Run()
```

Comment:
you shouldn't create kind inside `suite_test.go`, my intention is to use a shell script to start the kind setup and test. you may take a look at how `kind-setup.sh` being used in upstream.